sentinel-core/src/main/java/com/alibaba/csp/sentinel/util/ConfigUtil.java

```diff
@@ -107,20 +113,11 @@ private static Properties loadPropertiesFromClasspathFile(String fileName) {
 
         Properties properties = new Properties();
         for (URL url : list) {
-            try {
+            try (BufferedReader bufferedReader =
+                         new BufferedReader(new InputStreamReader(url.openStream(), Charset.forName(SentinelConfig.charset())))) {
```

Comment:
It seems that there's potential circular static dependency here: `SentinelConfig` -> `SentinelConfigLoader` -> `ConfigUtil` -> `SentinelConfig`, which could lead to initialization failure. We have to ensure that the `SentinelConfig` is not referenced by `ConfigUtil`.

Here we may need to provide a universal charset in `ConfigUtil`:

```java
private static final String CHARSET = System.getProperty("csp.sentinel.charset", "UTF-8");
```